client-app/src/Components/NewItemForm.tsx

```diff
@@ -111,77 +108,57 @@ const NewItemForm: React.FC = () => {
         }
     };
 
-    // Remove image from preview and form data
     const removeImage = (index: number) => {
         const updatedImages = formData.imageUpload.filter(
             (_, i) => i !== index,
         );
         const updatedPreviews = previews.filter((_, i) => i !== index);
-
         setFormData(prevState => ({
             ...prevState,
             imageUpload: updatedImages,
         }));
         setPreviews(updatedPreviews);
     };
 
-    // Handle input changes, including image uploads
     const handleChange = async (
         e: ChangeEvent<HTMLInputElement | HTMLSelectElement>,
     ) => {
-        const { name, type, value, files } = e.target as HTMLInputElement;
+        const { name, value } = e.target;
 
-        if (name === 'imageUpload' && files) {
-            const fileArray = Array.from(files);
-            const base64Images = await Promise.all(
-                fileArray.map(file => convertToBase64(file)),
+        if (name === 'itemType') {
+            setFormData(prevState => ({
+                ...prevState,
+                itemType: value,
+            }));
+        } else if (name === 'donorEmail') {
+            const selectedDonor = donorEmailOptions.find(
+                option => option.value === value,
+            );
+            setFormData(prevState => ({
+                ...prevState,
+                donorEmail: value,
+                donorId: selectedDonor?.id || null,
```

Comment:
@miabu-pashh `donorId` and `programId `are sent as strings, can you define them to be integers in the `FormData `please? On making this change, I am able to merge this PR